src/main/java/core/basesyntax/Texts.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Date;
+
+public interface Texts {
+
+    default void greeting() {
```

Comment:
@YaTut1901 Why default methods? I don't understand the advantages of this approach.